api/src/Service/ReportStatusService.php

```diff
@@ -688,6 +688,38 @@ public function getLifestyleState()
         }
     }
 
+ /**
```

Comment:
Nothing new here I've just moved the function up and by the other state check functions :)